project/frontend/src/ResultsMap.js

```diff
@@ -8,15 +8,27 @@ const mapStyle = {
 };
 
 export class MapContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    // TODO: change this to be the first match coordinates
+    this.state = { lat: 40.837, lng: -73.865 };
+    this.changeCenter = this.changeCenter.bind(this);
+  }
+
+  changeCenter() {
+    // TODO: change this to read off of the returned restaurants
+    this.setState({ lat: 37.419, lng: -122.078 });
+  }
+
   render() {
     return (
       <Map
         google={this.props.google}
         zoom={14}
         style={mapStyle}
-        initialCenter={{
-          lat: -1.2884,
-          lng: 36.8233,
+        center={{
+          lat: this.state.lat,
+          lng: this.state.lng,
```

Comment:
nit: after render pull these out `const { lat, lng } = this.state` and then here you can just do center = { lat, lng }

Reply:
quick question about this! if i add more states for my Map could I still do `const {lat,lng} = this.state` ? im adding more states in another branch and i'm wondering if it will affect this

Reply:
More fields to this state? Shouldn't affect that! Thats's just the equivalent of saying this.state.lat and this.state.lng